website/content/docs/templates/index.mdx

```diff
@@ -28,4 +28,4 @@ From version **1.7.0**, HCL2 becomes officially the preferred way to write Packe
 configuration(s).
 
 If you need help upgrading a JSON template to an HCL template, check
-our upgrade guide [here](/guides/hcl/from-json-v1).
+our upgrade guide [here](/guides/v1.6.x/hcl/from-json-v1).
```

Comment:
This guide has been replaced by an actual learn guide. So we can point there as opposed to a previous version of the docs. 
```suggestion
our upgrade guide [here](https://developer.hashicorp.com/packer/tutorials/configuration-language/hcl2-upgrade).
```